sdk/purview/azure-purview-sharing/samples/README.md

```diff
@@ -13,24 +13,29 @@ description: Samples for the Azure.Purview.Sharing client library.
 
 The following are code samples that show common scenario operations with the Azure Purview client library.
 
-* [sent_shares_examples.py](https://github.com/yamanwahsheh/azure-sdk-for-python/blob/yaman/share-v2-python-tests-and-samples/sdk/purview/azure-purview-sharing/samples/sent_shares_examples.py) - Examples of Sent Shares:
+* [sent_shares_examples.py](https://github.com/hopewelchmicrosoft/azure-sdk-for-python/blob/hope-new-branch/sdk/purview/azure-purview-sharing/samples/sent_shares_examples.py) - Examples of Sent Shares:
```

Comment:
This should point to the official repo.

Reply:
I agree. I've been stumped on how to do that for a while but I just realized, if I pointed it to the official repo, then once this is merged, those files I point to will have my changes so it shouldn't be a problem!